packages/devtools-kit/src/api/off.ts

```diff
@@ -1,5 +1,5 @@
-import { apiHooks } from './hook'
+import { instanceHooks } from './hook'
 
 export function remove() {
-  apiHooks.removeAllHooks()
```

Comment:
The current implementation mistakenly removes all hooks including the one-time registered hook (eg. DevToolsEvents.DEVTOOLS_STATE_UPDATED).